Makefile

```diff
@@ -112,11 +108,12 @@ lint: build staticcheck ## run all linters
 
 .PHONY: clean
 clean: ## clean temporary files and directories
-	go clean -i -cache -testcache
```

Comment:
-cache or -testcache flags remove entire go build cache.
https://github.com/golang/go/issues/53725 added warning about it which we silenced in #2217